cypress/e2e/sample_test_spec/SampleTestRequest.cy.ts

```diff
@@ -0,0 +1,83 @@
+import { SampleTestPage } from "pageobject/Sample/SampleTestCreate";
+import { PatientPage } from "pageobject/Patient/PatientCreation";
+import LoginPage from "pageobject/Login/LoginPage";
+
+describe("Sample Test", () => {
+  const sampleTestPage = new SampleTestPage();
+  const patientPage = new PatientPage();
+  const loginPage = new LoginPage();
+
+  const patientName = "Dummy Patient 11";
+  const sampleTestType = "BA/ETA";
+  const icmrCategory = "Cat 0";
+  const icmrLabel = "Test Icmr Label";
+  const doctorName = "Dr. John Doe";
+  const atypicalDetails = "Patient showing unusual symptoms";
+  const diagnosis = "Suspected respiratory infection";
+  const etiologyIdentified = "Bacterial infection suspected";
+  const differentialDiagnosis = "Possibly a viral infection";
+  const fastTrackReason =
+    "The patient has a high risk of complications and requires immediate testing.";
+
+  before(() => {
+    loginPage.loginAsDisctrictAdmin();
+    cy.saveLocalStorage();
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.clearLocalStorage(/filters--.+/);
+  });
+
+  it("should request a new sample test", () => {
+    // Ensure patient list API is loaded before proceeding
+    cy.awaitUrl("/patients");
+    patientPage.visitPatient(patientName);
+    cy.verifyAndClickElement("#patient-details", "Patient Details");
```

Comment:
```suggestion
    cy.verifyAndClickElement("#patient-details", "Patient Details");
```

- reusable the clickPatientDetails() function in the PatientConsultation.ts